mathics_django/web/templates/500.html

```diff
@@ -1,5 +1,6 @@
-{% extends "base.html" %}
-
 {% block content %}
-	<h1>Server error</h1>
-{% endblock content %}
\ No newline at end of file
+<h1>Server error</h1>
+<p>URI: {{request_path}}</p>
+<p>Sorry - no angry unicorn logos here. They are out to pasture.</p>
+{% endblock content %}
+{% extends "base.html" %}
```

Comment:
Why this page extends base HTML but 404 don't?

Reply:
When use that base, I get the useless logo header with the download, share, and load links, but without the images.